### DISCUSSION TITLE
User management: account deletion

### DISCUSSION THREAD

**Discussion by flefevre** [open-webui/open-webui#5817] [Ideas]
Dear all,
it will be good to be able to delete an account from the admin panel.
Thanks
François

**Comment by MicAnKa** [comment]
Hello!

You can already do this, as long as your account is an admin account, and the account you need to remove is a "user" account.

![image](https://github.com/user-attachments/assets/55ba42fd-2cd7-4ba1-8826-fb189f57e661)

Hope this helps.

**Reply by MicAnKa** [reply]
You can also delete "Pending" accounts

**Reply by NUSTHR** [reply]
why can not I see such buttons. I am in admin account

**Reply by NUSTHR** [reply]
![image](https://github.com/user-attachments/assets/bf9f8d71-ebbf-4925-9d8a-1767a1b8a0d9)

**Comment by flefevre** [comment]
Thanks you are right

**Comment by m20l22** [comment]
I suspect this does not delete a user from the database. The user was deleted from UI. I receive the following message when trying to recreate a user with the same e-mail address. I suspect it is still stored in the database even though it is not list by UI.

> Uh-oh! This email is already registered. Sign in with your existing account or choose another email to start anew.

**Comment by kmanwar89** [comment]
I created a secondary admin account that I'd like to delete, but it says this is restricted as a "security feature", which is understandable. Assuming I accept the risk, and have a backup admin account already configured, is there any way to bypass this security feature and perform the requested action? Thanks

**Comment by flefevre** [comment]
You have to switch it to normal user before?

**Reply by kmanwar89** [reply]
A normal user can't delete an Admin user's account, but an admin also cannot delete another admin's account, it seems.

**Comment by atol71** [comment]
Change the admin user to normal user and delete.
